### DISCUSSION TITLE
Wierd Pool Behavior

### DISCUSSION THREAD

**Discussion by cypher619** [Chia-Network/chia-blockchain#8358] [Support]
When setting up for pooling I accidentally made two plot NFTs.  I chose to use the second of the NFTs to connect to pool and plot.  First Plot NFT is set up for Self Pool with 0 plots.  Second Plot NFT is connected to pool with over 1k plots.

The wierd behavior is that I am recieving pool rewards to the rewards address off both Plot NFT's.  If you look up both rewards addresses at my pool you can see activity on both from the same node addresses.  I was only monitoring one of the Payout Addresses and thought that my chia farm was under performing.  I was seeing rewards deposits coming from both payout address but didn't understand why.

I plotted my plots with Madmax and I just double checked the keys I used to create the plots and they are the same as the Plot NFT that is connected to a pool.

Anyone else having this problem or did I just stumble on a bug?

**Comment by cypher619** [comment]
I've been monitoring my status and it appears the more active reward address belongs to the Plot NFT that has no plots associated with it.  80TB of drive space just arrived so I want to get this resolved before I go plotting on them

**Reply by loppefaaret** [reply]
are both plotnfts from the same key ?

**Reply by cypher619** [reply]
If you mean same key as in private keys then yes.

**Comment by BrandtH22** [comment]
@cypher619 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

Note that the payouts from pools are determined by the farmers pool payout address (or directly via the connected pool) and are not effected by the plotNFT key used to create the plots. It is likely that the payout address settings need to be aligned for each.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
